### ISSUE TITLE
[Bug][user.go] Uploading users.csv fails with run time error 

### ISSUE THREAD

**Issue by yacc** [apache/devlake#7219]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

Using curl to upload the users.csv file with the template users.csv file downloaded from Devlake is generating a run time error.

### What do you expect to happen

I expected to have the data from the users.csv file to be uploaded in the db

### How to reproduce

Running 
```
curl 'http://127.0.0.1:4000/api/rest/plugins/org/users.csv' -X 'PUT' -H "Authorization: Bearer $DEVLAKE_API_KEY" --form 'file=@"./users.csv"'
```
with 
```
➜  devlake cat users.csv
Id,Name,Email,TeamIds
1,Tyrone K. Cummings,TyroneKCummings@teleworm.us,1;2
2,Dorothy R. Updegraff,DorothyRUpdegraff@dayrep.com,3
```

### Anything else

Here's the logs:
```log
2024-03-26 10:23:09 [GIN] 2024/03/26 - 09:23:09 | 500 |    8.590084ms |      172.18.0.5 | PUT      "/rest/plugins/org/users.csv"
2024-03-26 10:23:09 
2024-03-26 10:23:09 
2024-03-26 10:23:09 2024/03/26 09:23:09 [Recovery] 2024/03/26 - 09:23:09 panic recovered:
2024-03-26 10:23:09 runtime error: invalid memory address or nil pointer dereference
2024-03-26 10:23:09 /usr/local/go/src/runtime/panic.go:260 (0x9955e7)
2024-03-26 10:23:09 /usr/local/go/src/runtime/signal_unix.go:837 (0x9ad803)
2024-03-26 10:23:09 /app/plugins/org/api/user.go:91 (0xffff46fa2328)
2024-03-26 10:23:09 /app/server/api/router.go:135 (0x149d547)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 (0x149b8cb)
2024-03-26 10:23:09 /app/server/api/middlewares.go:95 (0x149b8a8)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 (0x149bcfb)
2024-03-26 10:23:09 /app/server/api/middlewares.go:117 (0x149bbd4)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 (0x114a97b)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/recovery.go:102 (0x114a95c)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 (0x1149c5b)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/logger.go:240 (0x1149c38)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 (0x1148b5b)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/gin.go:620 (0x1148884)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/gin.go:587 (0x11485ff)
2024-03-26 10:23:09 /app/server/api/middlewares.go:126 (0x149bcbf)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 (0x114a97b)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/recovery.go:102 (0x114a95c)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 (0x1149c5b)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/logger.go:240 (0x1149c38)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174 (0x1148c4f)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/gin.go:656 (0x1148c30)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/gin.go:649 (0x11489ff)
2024-03-26 10:23:09 /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/gin.go:576 (0x1148483)
2024-03-26 10:23:09 /usr/local/go/src/net/http/server.go:2936 (0xc798cf)
2024-03-26 10:23:09 /usr/local/go/src/net/http/server.go:1995 (0xc753c7)
2024-03-26 10:23:09 /usr/local/go/src/runtime/asm_arm64.s:1172 (0x9c9523)
2024-03-26 10:23:09 
```

### Version

v1.0.0-beta1@60faf14

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Startrekzky** [comment]
Adding more contexts from the community discussion:
![image](https://github.com/apache/incubator-devlake/assets/14050754/1e0ab793-4564-4582-9d75-7d01b33a2c79)

**Comment by klesh** [comment]
Please open your `config-ui` and migrate your database first.

**Comment by yacc** [comment]
@klesh Thanks for your help, I don't see a way to migrate the db in the `config-ui` and I can't find a reference to this in the manual. Can you give more details?

**Comment by klesh** [comment]
@yacc It should pop up when there are migration scripts to be executed.
So, no prompt when you open the `config-ui`?

**Comment by klesh** [comment]
@yacc I see.
I see you checked the following checkbox, are you working on it?
![image](https://github.com/apache/incubator-devlake/assets/61080/a8f77ac3-775e-46db-b3dc-ca0aa5da7a5f)

**Comment by yacc** [comment]
@klesh I'm willing to, but it's gonna take a while, I don't have a lot of bandwidth these days

**Comment by klesh** [comment]
@yacc Thanks, take you time, no rush.

**Comment by fchenebault** [comment]
I had the same issue and it's now fixed after v1.0.0-beta4.
Thanks a lot!
